piperider_cli/profiler/schema.json

```diff
@@ -443,6 +449,79 @@
         }
       }
     },
+    "metric": {
+      "title": "Metric",
+      "type": "object",
+      "required": [
+        "name",
+        "label",
+        "description",
+        "grain",
+        "dimensions",
+        "headers",
+        "data"
+      ],
+      "additionalProperties": false,
+      "properties": {
```

Comment:
would be good to add `properties.<field>.description` so that we remember what each field means

Reply:
sure, I will add a `description` for each field.